meson_options.txt

```diff
@@ -1,3 +1,4 @@
 option('krb5', type : 'feature',
   description : 'Build with Kerberos support',
+  value : 'disabled',
```

Comment:
Can't say I like this. The whole reason this is a feature and not a bool is to build krb5 when available. Is there a reason it shouldn't be built?

Reply:
@neheb, please make it clear that this is a question for @namjaejeon and @hclee.
Why is Kerberos 5 support disabled by default in the autotools build?

Reply:
Regardless of the reason, the point of this commit is to have parity between the autotools and meson builds. If it is desirable that Kerberos 5 is enabled by default when it is available, then a later commit can make the change for both build systems.

Reply:
That's why reason is that kerberos in ksmbd kernel is disable by default.